src/logic/operate.js

```diff
@@ -0,0 +1,22 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const figureOne = Big(numberOne);
+  const figureTwo = Big(numberTwo);
+  if (operation === '+') {
+    return figureOne.plus(figureTwo).toString();
+  }
+  if (operation === '-') {
+    return figureOne.minus(figureTwo).toString();
+  }
+  if (operation === 'x') {
+    return figureOne.times(figureTwo).toString();
+  }
+  if (operation === '÷') {
+    return figureOne.div(figureTwo).toString();
+  }
+  if (operation === '%') {
+    return figureOne.mod(figureTwo).toString();
```

Comment:
**[Required]**  The % operator should return decimal number as mentioned by the previous TSE. I believe it is not a modulo operator. Please have a look at the [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone4/performOperate.png)